hnn_core/tests/test_network.py

```diff
@@ -510,6 +510,30 @@ def test_network_drives_legacy():
                                                  n_bursty_sources)
 
 
+def get_expected_connectivities(net, src_types='all'):
```

Comment:
just make this a private function ... it's not user facing

Reply:
Sure will do